docs: se traduce el archivo deprecations.md al español

devacran opened the pull request:
Se traduce el archivo deprecations.md al español Fixes
Se actualizan los links asociados al documento, en los siguientes archivos:
testing-services.en.md
testing-services.md
ng_module.ts

## Lista de Verificación del PR
Comprueba si tu PR cumple los siguientes requisitos:

- [x] El mensaje de commit esta conforme con [nuestras reglas](https://github.com/angular-hispano/angular/blob/master/CONTRIBUTING.md#-formato-para-el-mensaje-de-los-commits)
- [x] Probe los cambios que agregué (arreglo de bugs / funcionalidades)
- [x] Revisé previamente las traducciones o cambios de contenido
- [x] Consulté el [diccionario de términos](https://github.com/angular-hispano/angular/blob/master/aio/diccionario-de-términos.md) en español
- [x] He creado dos archivos con la extensión correspondiente(.en.md para el archivo en inglés y .md para el Archivo en español)
- [x] He enlazado el commit con el issue correspondiente <!-- ejemplo Fixes #X  -->


## Tipo de PR
¿Qué tipo de cambio introduce este PR?

<!-- Marca con una "x" las opciones que aplican. -->

- [ ] Bugfix
- [ ] Funcionalidad
- [ ] Actualización de el estilo del código (formato, variables locales)
- [ ] Refactorización (no cambios en la funcionalidad, no cambios en el api)
- [ ] Cambios relacionados al build
- [ ] Cambios relacionados al CI (Integración continua)
- [x] Cambios en el contenido de la documentación
- [ ] Cambios en la aplicación / infraestructura de angular.io
- [ ] Otro... Por favor describe la:

## ¿Cuál es el comportamiento actual?
Archivo deprecations.md está en inglés
<!-- Describe el comportamiento actual que está modificando o vincule a un problema relevante.
-->


## ¿Cuál es el nuevo comportamiento?
Archivo deprecations.md esta traducido al español
<!--
Ejemplo: Archivo en inglés traducido al español
-->

devacran committed:
docs: se traduce el archivo deprecations.md al español fixes(#187)

Se actualizan los links asociados al documento, en los siguientes archivos:
testing-services.en.md
testing-services.md
ng_module.ts

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/angular-hispano/angular?pullRequest=362) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/angular-hispano/angular?pullRequest=362) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/angular-hispano/angular?pullRequest=362) it.</sub>

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -1,26 +1,22 @@
 
-Angular strives to balance innovation and stability.
-Sometimes, APIs and features become obsolete and need to be removed or replaced so that Angular can stay current with new best practices, changing dependencies, or changes in the (web) platform itself.
+Angular hace esfuerzos por equilibrar innovación y estabilidad. En ocasiones API’s y funcionalidades se vuelven obsoletas y necesitan ser eliminadas o reemplazadas para que Angular pueda mantenerse actualizado con nuevas mejores prácticas, cambios en las dependencias o cambios en la plataforma (web) en sí.
```
Corregir "API’s" => "APIs" creo no es necesario el apostrofe

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -1,26 +1,22 @@
-This guide contains a summary of all Angular APIs and features that are currently deprecated.
+Para hacer la transición lo más fácil posible, API’s y funcionalidades se vuelven obsoletas por un periodo de tiempo antes de eliminarlas. Esto da tiempo para que actualice sus aplicaciones a las nuevas API’s y a las mejores prácticas más recientes.
 
+Esta guía contiene un resumen de todas las API’s y funcionalidades de angular que están en desuso actualmente.
```
Corregir "funcionalidades de angular" => "funcionalidades de Angular", ya que es nombre propio

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -1,26 +1,22 @@
-Features and APIs that were deprecated in v6 or earlier are candidates for removal in version 9 or any later major version. For information about Angular's deprecation and removal practices, see [Angular Release Practices](guide/releases#deprecation-practices "Angular Release Practices: Deprecation practices").
-
-For step-by-step instructions on how to update to the latest Angular release, use the interactive update guide at [update.angular.io](https://update.angular.io).
+Para informacion acercacomo actualizar Angular a la última versión, visita la guía interactiva en [update.angular.io](https://update.angular.io).
```
Corregir "informacion acercacomo" => "información acerca como"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
-
-This section contains a complete list all of the currently-deprecated APIs, with details to help you plan your migration to a replacement.
-
+| Area                          | API or Feature                                                                                | Probalemente Eliminada    |
```
Corregir "Area" => "Área"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
 
+### @angular/core
 
+| API                                                                     | Replazo                                                               | Anuncio de su desuso | Notas                                                                                                                                                                                                     |
```
Corregir "Replazo" => "Remplazo"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
 <div class="alert is-helpful">
 
-Tip: In the [API reference section](api) of this doc site, deprecated APIs are indicated by ~~strikethrough.~~ You can filter the API list by [**Status: deprecated**](api?status=deprecated).
+Tip: En la [Seccion de referencia de API's](api) de este documento, las API's obsoletas se muestran ~~tachadas.~~ Puedes filtrar la lista de API's por [**Estado: en desuso**](api?status=deprecated).
```
Corregir "Seccion" => "Sección"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
- in the Component Styles guide.
+El combinado de descendientes shadow-dom-piercing está obsoleto y su soporte está siendo [retirado de la mayoría de los navegadores y herramientas.](https://developers.google.com/web/updates/2017/10/remove-shadow-piercing). Como resultado, en la versión 4 se volvió obsoleto el soporte in Angular para todos los 3 `/deep/`, `>>>` and `::ng-deep`. Hasta su eliminación, es preferido usar `::ng-deep`  ...
 
+Para mas informacion, visita [/deep/, >>>, and ::ng-deep](guide/component-styles#deprecated-deep--and-ng-deep "Component Styles guide, Deprecated deep and ngdeep")
```
Corregir "mas informacion" => "más información"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
 ### ReflectiveInjector
 
-In v5, Angular replaced the `ReflectiveInjector` with the `StaticInjector`. The injector no longer requires the Reflect polyfill, reducing application size for most developers.
+En la version 5, Angular remplazo `ReflectiveInjector` por `StaticInjector`. El inyector ya no requiere el polyfill Reflect, lo que reduce el tamaño de la aplicación para la mayoría de los desarrolladores.
```
Corregir "version" => "versión"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
 <div class="alert is-helpful">
 
-**Declaration syntax**: It's important to follow the route declaration syntax `loadChildren: () => import('...').then(m => m.ModuleName)` to allow `ngc` to discover the lazy-loaded module and the associated `NgModule`. You can find the complete list of allowed syntax constructs [here](https://github.com/angular/angular-cli/blob/a491b09800b493fe01301387fa9a025f7c7d4808/packages/ngtools/webpack/src ...
+**Sintaxis de declaracion**: Es importante seguir la sintaxis de declaración de ruta `loadChildren: () => import ('...'). Luego (m => m.ModuleName)` para permitir que `ngc` descubra el módulo de carga diferida y el` NgModule` asociado. Puede encontrar la lista completa de construcciones de sintaxis permitidas [aqui](https://github.com/angular/angular-cli/blob/a491b09800b493fe01301387fa9a025f7c7d4 ...
```
Corregir "declaracion" => "declaración"

nicobytes commented on aio/content/guide/deprecations.md:
````diff
@@ -32,546 +28,546 @@ v8 - v11
 ```
 
-However, in practice, Angular simply ignores two-way bindings to template variables. Starting in version 8, attempting to write to template variables is deprecated. In a future version, we will throw to indicate that the write is not supported.
+Sin embargo, en la práctica, Angular simplemente ignora los enlaces bidireccionales a las variables de plantilla. A partir de la versión 8, el intento de escribir en variables de plantilla ha quedado obsoleto. En una versión posterior, se lanzara una alerta para indicar que la escritura no es compatible.
````
Corregir "lanzara " => "lanzará "

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
-All of the `wtf*` APIs are deprecated and will be removed in a future version.
+### `wtfStartTimeRange` y todas las APIs `wtf*`
+
+Todo acerca de las APIs `wtf*` ha quedado obsoleto y sera eliminado en la version posterior.
```
Corregir "sera" => "será" y "version" => "versión"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
+La versión 9 de Angular ha dejado en desuso `ModuleWithProviders` sin un tipo explícitamente genérico, donde el tipo genérico se refiere al tipo de` NgModule`.
+En una versión futura de Angular, el genérico ya no será opcional.
+Más sobre el texto fuente
+Si está utilizando el CLI, "ng update" debería [migrar el codigo automaticamente](guide/migration-module-with-providers).
```
Corregir "el codigo automaticamente" => "el código automáticamente"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
-- Explicitly call [`ChangeDetectorRef.detectChanges()`](api/core/ChangeDetectorRef#detectchanges) to force the update.
+Si confía en el comportamiento de que la misma instancia del objeto debería causar la detección de cambios, existen dos opciones:
+
+- Clonar el valor resultante para que lenga una nueva identidad.
```
Corregir "lenga" => "tenga"

nicobytes commented on aio/content/guide/deprecations.md:
```diff
@@ -601,20 +598,23 @@ In practical terms, the `package.json` of all `@angular` packages has changed in
+
+### Sanitizacion de estilo los enlaces de `[style]` y `[style.prop]`
+
+Angular solia sanitizar los enlaces de `[style]` y `[style.prop]` para evitar que se inserte código malicioso a través de las expresiones `javascript:` en las entradas `url ()` de CSS. Sin embargo, la mayoría de los navegadores modernos ya no admiten el uso de estas expresiones, por lo que la sanitizacion solo se mantuvo por el bien de IE 6 y 7. Dado que Angular no admite IE 6 o 7 y la sanitizaci ...
```
Corregir "solia " => "solía "

devacran committed:
docs: resolve requested changes

devacran committed:
docs: resolucion de los issues de traduccion del archivo deprecations.md fixes(#187)

Se hizo la traduccion del archivo deprecations.md y de los links relacionados a este.

Fix #182

nicobytes commented:
@devacran Por favor agrega el pie de página de commit como se menciona en [https://github.com/angular-hispano/angular/blob/master/CONTRIBUTING.md#mensaje-del-footer-del-commit](https://github.com/angular-hispano/angular/blob/master/CONTRIBUTING.md#mensaje-del-footer-del-commit), por ejemplo, "Fixes #0". Esto con el objetivo que cuando se haga merge se cierre automáticamente el issue.

Lo puedes arreglar con `git commit --amend` y `git push --force-with-lease`

Splaktar commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
-This type represents an `NgModule` along with additional providers.
-Angular version 9 deprecates use of `ModuleWithProviders` without an explicitly generic type, where the generic type refers to the type of the `NgModule`.
-In a future version of Angular, the generic will no longer be optional.
+### `ModuleWithProviders` tipo sin generico
```
Para que los enlaces sigan funcionando, podemos hacer lo siguiente:
https://github.com/angular-hispano/angular/pull/354/files#diff-6c2f6d60a2b618e3d2cb9d69fd865bb391793c0c084dcbc6d43f59a5e8d085afR22

Queremos mantener estos PRs pequeños y no romper muchas otras cosas, por lo que queremos mantener estos fragmentos de enlaces en inglés. Para el PR tuyo, significó cambiar 3 archivos adicionales, para algunos otros requiere aún más, así que lo estamos evitando ya que simplifica el proceso.

Splaktar commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
-offloading CPU-intensive, non-critical work needed for initial rendering (such as in-memory search
-and image processing). Learn more in the
-[guide to Using Web Workers with the Angular CLI](guide/web-worker).
+`@angular/platform-webworker` se introdujo en la versión 2 de Angular como un experimento del aprovechamiento de la Arquitectura de renderizado de Angular para ejecutar una aplicación web completa en un [web worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API). Hemos aprendido bastante
```
```suggestion
`@angular/platform-webworker` se introdujo en la versión 2 de Angular como un experimento del aprovechamiento de la Arquitectura de renderizado de Angular para ejecutar una aplicación web completa en un [web worker](https://developer.mozilla.org/es/docs/Web/API/Web_Workers_API). Hemos aprendido bastante
```

Splaktar commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,546 @@ v8 - v11
 
+A partir de la versión 8 de Angular, todas las APIs de `platform-webworker` son obsoletas.
+Esto incluye ambos paquetes: `@angular/platform-webworker` y
+`@angular/platform-webworker-dynamic`.
```
```suggestion
`@angular/platform-webworker-dynamic`.

```

Splaktar reviewed (changes requested):
¡Gracias por el trabajo en esta gran pieza de traducción!

nicobytes commented:
Hola @devacran creo quedan pendientes los últimos ajustes que se solicitaron para poder proceder.

devacran commented:
> Hola @devacran creo quedan pendientes los últimos ajustes que se solicitaron para poder proceder.

Perdon jeje no habia mirado, ya hice cambios!

Splaktar commented on aio/content/guide/deprecations.md:
```diff
@@ -32,546 +28,547 @@ v8 - v11
+
+Anteriormente, el array `entryComponents` en la definición de` NgModule` se usaba para decirle al compilador qué componentes se crearían e insertarían dinámicamente. Con Ivy, esto ya no es un requisito y el array `entryComponents` se puede eliminar de las declaraciones de módulos existentes. Lo mismo se aplica al token de inyección `ANALYZE_FOR_ENTRY_COMPONENTS`.
+
+{@a modulewithproviders-type-without-a-generic}
```
Esto debe quedar como `{@a moduleWithProviders}`.

Splaktar reviewed (commented):
Perdón por la larga demora aquí. Se ha invertido mucho trabajo en este PR, por lo que estaría feliz de conseguirlo. Parece que solo se necesita un ajuste adicional para evitar romper enlaces.
